crates/macros/src/config/setting.rs

```diff
@@ -3,15 +3,22 @@ use crate::utils::extract_comment;
 use darling::FromAttributes;
 use proc_macro2::{Ident, TokenStream};
 use quote::{quote, ToTokens};
-use syn::{Expr, ExprPath, Field, Type};
+use syn::{Expr, ExprPath, Field, Meta, Type};
+
+pub fn preserve_str_literal(meta: &Meta) -> darling::Result<Option<Expr>> {
```

Comment:
The plan is to make this function (without the `Option` in the return type) available as part of the `darling` crate; see TedDriggs/darling#235 for that.

You can achieve the desired result here by adding `map = "Some"` to the attribute that is consuming this function (see comment suggestion below)

Reply:
Awesome, thanks.